code/__DEFINES/traits.dm

```diff
@@ -114,6 +114,30 @@
 #define HAS_TRAIT_FROM_ONLY(target, trait, source) (target.status_traits?[trait] && (source in target.status_traits[trait]) && (length(target.status_traits[trait]) == 1))
 #define HAS_TRAIT_NOT_FROM(target, trait, source) (target.status_traits?[trait] && (length(target.status_traits[trait] - source) > 0))
 
+/// Define for adding multiple traits to a target via a list. Must have a common source and target.
+#define ADD_TRAIT_LIST(target, list_of_traits, source) \
```

Comment:
I don't think we are gaining anything from these being macros unless we can prove we get a sizable performance benefit from them, please make them procs in a `__HELPERS/traits.dm` or something (since I think `__DEFINES` should only have defines)